sys/netinet6/frag6.c

```diff
@@ -57,6 +57,7 @@ __FBSDID("$FreeBSD$");
 #include <net/netisr.h>
 #include <net/route.h>
 #include <net/vnet.h>
+#include <net/rss_config.h>
```

Comment:
does this not have to be guarded with a define?

Reply:
It does, I'll make the appropriate changes